python_binding/lsm/_client.py

```diff
@@ -1427,3 +1427,320 @@ def volume_ident_led_clear(self, volume, flags=FLAG_RSVD):
+        return self._tp.rpc('volume_cache_info', _del_self(locals()))
+
+    @_return_requires(None)
+    def volume_physical_disk_cache_set(self, volume, pdc, flags=FLAG_RSVD):
+        """
+        lsm.Client.volume_physical_disk_cache_set(self, volume, pdc,
+                                                  flags=lsm.Client.FLAG_RSVD)
+
+        Version:
+            1.3
+        Usage:
+            Change the setting of RAM physical disk cache of certain volume.
+            On some product(like HP SmartArray), this action will be effective
+            at system level which means even you are requesting a change
+            on certain volume, it still impact on all other volumes on the same
+            controller(system).
+        Parameters:
+            volume (Lsm.Volume)
+                The lsm.Volume instance.
+            pdc (int)
+                lsm.Volume.PHYSICAL_DISK_CACHE_ENABLED
+                    Enable physical disk cache.
+                lsm.Volume.PHYSICAL_DISK_CACHE_DISABLED
+                    Disable physical disk cache
+            flags (int, optional):
+                Reserved for future use. Should be set as lsm.Client.FLAG_RSVD
+        Returns:
+            N/A
+        SpecialExceptions:
+            LsmError
+                ErrorNumber.NO_SUPPORT
+                    You might also get NO_SUPPORT error when trying
+                    to change SSD physical disk cache on MegaRAID.
+                ErrorNumber.NOT_FOUND_VOLUME
+        Capability:
+            lsm.Capabilities.VOLUME_PHYSICAL_DISK_CACHE_SET
+                Allow changing physical disk cache.
+            lsm.Capabilities.VOLUME_PHYSICAL_DISK_CACHE_SET_SYSTEM_LEVEL
+                Indicate this action will change system settings which
+                is effective on all volumes in this system.
+                For example, on HP SmartArray, physical disk cache
+                setting is a controller level setting.
+        """
+        if (pdc != Volume.PHYSICAL_DISK_CACHE_ENABLED) and \
+           (pdc != Volume.PHYSICAL_DISK_CACHE_DISABLED):
+            raise LsmError(ErrorNumber.INVALID_ARGUMENT,
+                           "Argument pdc should be "
+                           "Volume.PHYSICAL_DISK_CACHE_ENABLED or "
+                           "Volume.PHYSICAL_DISK_CACHE_DISABLED")
+
+        return self._tp.rpc('volume_physical_disk_cache_set',
+                            _del_self(locals()))
+
+    @_return_requires(None)
+    def volume_write_cache_policy_set(self, volume, wcp, flags=FLAG_RSVD):
+        """
+        lsm.Client.volume_write_cache_policy_set(self, volume, wcp,
+                                                 flags=lsm.Client.FLAG_RSVD)
+
+        Version:
+            1.3
+        Usage:
+            Change the RAM write cache policy on certain volume.
+            If lsm.Capabilities.VOLUME_WRITE_CACHE_POLICY_SET_IMPACT_READ
+            is supported(e.g. HP SmartArray), the changes on write cache policy
```

Comment:
Just change all the HP mentions to HPE in this pull if you could. I'll submit a pull when I can to update mentions as unintrusively as possible. :)